src/wallet/wallet_rpc_server.cpp

```diff
@@ -2425,7 +2425,7 @@ namespace tools
     catch (const tools::error::not_enough_outs_to_mix& e)
     {
       er.code = WALLET_RPC_ERROR_CODE_NOT_ENOUGH_OUTS_TO_MIX;
-      er.message = e.what();
+      er.message = e.what() + " Please use sweep_dust.";
```

Comment:
I get a compilation error here:
```
... error: invalid operands of types ‘const char*’ and ‘const char [24]’ to binary ‘operator+’
       er.message = e.what() + " Please use sweep_dust.";
```
I fixed it with:
```
er.message = e.what() + std::string(" Please use sweep_dust.");
```

Reply:
Thanks!